backend/handler/db.js

```diff
@@ -81,11 +121,13 @@
 		const collection = await Handler.getCollection(database, colName);
 		// NOTE: documentation states it should be { projection, sort } so when we upgrade we may have to change.
 		//       Also: projection stops working if you pass in a sort with empty obj.
-		if(sort) {
-			projection.sort = sort;
+		const options = { projection };
```

Comment:
remove comment above as it's addressed?

Reply:
also in `find`,  could we potentially rewrite it to be similar to findOne?

I had to write it that way because of the old API didn't work otherwise. be good if the way we access their api is consistent.